src/hoi4_world/countries/hoi4_country_converter.cpp

```diff
@@ -319,33 +319,39 @@ std::map<std::string, int> NormalizeIdeologySupport(const std::map<std::string,
       adjusted_ideology_support.emplace(ideology, raw_value - lowest_support);
    }
 
-   const float total_support = std::accumulate(adjusted_ideology_support.begin(),
+   const float total_raw_support = std::accumulate(adjusted_ideology_support.begin(),
        adjusted_ideology_support.end(),
        0.F,
        [](float a, const auto& b) {
           return a + b.second;
        });
 
    std::map<std::string, int> ideology_support;
-   if (total_support != 0.F)
+   if (total_raw_support != 0.F)
    {
-      const float normalization_factor = 100.F / total_support;
+      const float normalization_factor = 100.F / total_raw_support;
       for (const auto& [ideology, raw_value]: adjusted_ideology_support)
       {
-         int value = static_cast<int>(std::round(raw_value * normalization_factor));
+         int value = static_cast<int>(raw_value * normalization_factor);
          ideology_support.emplace(ideology, value);
       }
    }
    else
    {
-      const int normalization_factor =
-          static_cast<int>(std::round(100.F / static_cast<float>(adjusted_ideology_support.size())));
+      const int normalization_factor = static_cast<int>(100.F / static_cast<float>(adjusted_ideology_support.size()));
       for (const auto& ideology: adjusted_ideology_support | std::views::keys)
       {
          ideology_support.emplace(ideology, normalization_factor);
       }
    }
 
+   ideology_support["neutrality"] = 0;
+   const int total_support =
+       std::accumulate(ideology_support.begin(), ideology_support.end(), 0, [](int a, const auto& b) {
+          return a + b.second;
+       });
+   ideology_support["neutrality"] = 100 - total_support;
```

Comment:
This sounds so ready to be 100-101 = -1. :D

Reply:
If I were still rounding, yeah. But going back to truncation should make it impossible.